quickref/qsharp-quick-reference.tex

```diff
@@ -663,17 +663,15 @@
 
   \begin{keysref}{Qubit Allocation}
     Allocate a register \newline of $N$ qubits 
-                         & \texttt{using (\emph{reg} = Qubit[\emph{N}]) \{} \newline
-                           \texttt{\hphantom{....}//} Qubits in \texttt{\emph{reg}} start in $\ket{0}$. \newline
-                           \texttt{\hphantom{....}...} \newline
-                           \texttt{\hphantom{....}//} Qubits must be returned to $\ket{0}$. \newline
-                           \texttt{\}} \\
-    Allocate one qubit   & \texttt{using (\emph{one} = Qubit()) \{ ... \} } \\
+                         & \texttt{use \emph{reg} = Qubit[\emph{N}]; } \newline
+                           \texttt{//} Qubits in \texttt{\emph{reg}} start in $\ket{0}$. \newline
+                           \texttt{...} \newline
+                           \texttt{//} Qubits must be returned to $\ket{0}$. \\
+    Allocate one qubit   & \texttt{use \emph{one} = Qubit();  ...  } \\
```

Comment:
Can you please move `...` to a new line? It looks a bit odd on the same line with `use`